benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/WorkerServiceGrpc.java

```diff
@@ -392,11 +396,32 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
     }
   }
 
-  private static final class WorkerServiceDescriptorSupplier implements io.grpc.protobuf.ProtoFileDescriptorSupplier {
+  private static class WorkerServiceFileDescriptorSupplier
```

Comment:
I would prefer if this class could be final, since every other class in this generate code is final too.

What do you think about making this final, making a static instance on the `MethodDescriptorSupplier`, and having `MethodDescriptorSupplier` implement the interfaces here?  


(non final classes, even private ones can be mocked by spying instances of them.  This was the cause of a recent test breakage)